lib/src/components/multi_date_content.dart

```diff
@@ -324,10 +324,15 @@ class _PartDayDraggableEventState extends State<PartDayDraggableEvent> {
       return;
     }
 
-    final adjustedOffset = _pointerToWidgetTopCenter(_lastOffset!);
-    final geometry = _findGeometry(context, adjustedOffset);
-    widget.onDragCanceled?.call(geometry.key, _wasMoved);
-    _resetState();
+    if (mounted) {
+      final adjustedOffset = _pointerToWidgetTopCenter(_lastOffset!);
+      final geometry = _findGeometry(context, adjustedOffset);
+      widget.onDragCanceled?.call(geometry.key, _wasMoved);
+      _resetState();
+    } else {
+      widget.onDragCanceled?.call(null, _wasMoved);
+      _resetState();
+    }
```

Comment:
What problem were you seeing with the previous code?

With the new code, if this widget is created as `PartDayDraggableEvent.forGeometryKeys(…)`, the call to `onDragCanceled` will throw an exception since it ensures that the geometry key is not `null`

Reply:
Thank you for taking the time to review the code. I apologize for the confusion caused by the incorrect order. I have realized that the letter "Ch" is specific to my language, Czech.

I made this change to implement the drag-and-drop functionality in my app, using the example app as a guide. However, there have been instances where the PartDayDraggableEvent becomes unmounted, and the onDragCanceled function crashes due to the unmounted state at `_pointerToWidgetTopCenter(_lastOffset!)`. As a result, the logic to remove the event overlay is not executed.

To address this issue, I have made a change that ensures the onDragCanceled function is called even after the widget is mounted, which removes the overlay in my app.

But I see that it conflicts with other functionality. What do you think I should do?

Reply:
A stacktrace:
```
Fatal Exception: io.flutter.plugins.firebase.crashlytics.FlutterError: Null check operator used on a null value. Error thrown Instance of 'ErrorDescription'.
       at State.context(framework.dart:954)
       at _PartDayDraggableEventState._findRenderBox(multi_date_content.dart:274)
       at _PartDayDraggableEventState._pointerToWidgetTopCenter(multi_date_content.dart:334)
       at _PartDayDraggableEventState._onDragCanceled(multi_date_content.dart:327)
       at AsyncValueX.maybeWhen.<fn>(common.dart:624)
       at _DraggableState._startDrag.<fn>(drag_target.dart:525)
       at _DragAvatar.finishDrag(drag_target.dart:897)
       at _DragAvatar.end(drag_target.dart:801)
       at MultiDragPointerState._up(multidrag.dart:158)
       at MultiDragGestureRecognizer._handleEvent(multidrag.dart:266)
       at PointerRouter._dispatch(pointer_router.dart:98)
       at PointerRouter._dispatchEventToRoutes.<fn>(pointer_router.dart:143)
       at _LinkedHashMapMixin.forEach(dart:collection)
       at PointerRouter._dispatchEventToRoutes(pointer_router.dart:141)
       at PointerRouter.route(pointer_router.dart:127)
       at GestureBinding.handleEvent(binding.dart:460)
       at GestureBinding.dispatchEvent(binding.dart:440)
       at RendererBinding.dispatchEvent(binding.dart:336)
       at GestureBinding._handlePointerEventImmediately(binding.dart:395)
       at GestureBinding.handlePointerEvent(binding.dart:357)
       at GestureBinding._flushPointerEventQueue(binding.dart:314)
       at GestureBinding._handlePointerDataPacket(binding.dart:295)```

Reply:
Sorry for the delay! I'll leave that up to you – go ahead if you want to, otherwise, I'll implement that change before the next release :)